### PR TITLE
fixed: OC: IE 11 and Edge Legacy - can't load external data(add polyfill for CSS.escape)

### PR DESCRIPTION
**Pull request by theywa**
#388

### CODE REVIEW

**Review by MartijnR — COMMENTED** [review]
Hi @theywa,

Just wanted to check if you confirmed that polyfill.io doesn't have this polyfill.

**Comment by theywa** [comment]
@MartijnR ~ I'm not sure how to check on polyfill.io but the current ie11-polyfill.min.js used and obscure-ie11-polyfills.js did not contain that polyfill

**Comment by pbowen-oc** [comment]
@MartijnR - Is anything else needed to merge this PR from @theywa?

**Review by MartijnR — APPROVED** [review]

**Comment by MartijnR** [comment]
> @MartijnR - Is anything else needed to merge this PR from @theywa?

No, thanks for reminding. I actually though I had merged it this morning, but apparently got interrupted midway and then forgot...

File: package-lock.json [hunk 1]
```diff
@@ -9,201 +9,201 @@
             "resolved": "https://registry.npmjs.org/@babel/cli/-/cli-7.10.5.tgz",
             "integrity": "sha512-j9H9qSf3kLdM0Ao3aGPbGZ73mEA9XazuupcS6cDGWuiyAcANoguhP0r2Lx32H5JGw4sSSoHG3x/mxVnHgvOoyA==",
             "requires": {
-                "chokidar": "^2.1.8",
```
Comment (MartijnR) [inline package-lock.json:12]: @theywa, next time no changes in package-lock.json please (other than for modules in package.json that you changed). It makes merging a pain.

**Review by MartijnR — APPROVED** [review]

File: Gruntfile.js [hunk 2]
```diff
@@ -91,7 +91,8 @@ module.exports = grunt => {
             'polyfill-ie11': {
                 command: [
                     'mkdir -p public/js/build && curl "https://polyfill.io/v3/polyfill.min.js?ua=ie%2F11.0.0&features=es2015%2Ces2016%2Ces2017%2Ces2018%2Cdefault-3.6%2Cfetch%2CNodeList.prototype.forEach" -o "public/js/build/ie11-polyfill.min.js"',
```
[the remark this answers is not in the document]
Comment (theywa) [reply Gruntfile.js:93]: I will, thank you

File: package-lock.json (hunk 1 above)
Comment (theywa) [reply package-lock.json:12]: sorry about that, I will keep that in mind.

**Pull request merged**